scripts/in_container/run_generate_constraints.py

```diff
@@ -384,7 +384,6 @@ def generate_constraints_pypi_providers(config_params: ConfigParams) -> None:
     run_command(
         cmd=[
             *config_params.get_install_command,
-            "-e",
```

Comment:
We need to remove editable installation here - when installing providers from PyPI - otherwise installing FAB provider from PyPI conflicts with non-released-yet FAB provider when they use different FAB.